### ISSUE TITLE
Ventoy.json and ventoy_grub.cfg missing from installation

### ISSUE THREAD

**Issue by andybot89** [ventoy/Ventoy#846]
On one of my drives using version 0.69, I was able to modify the "Ventoy.json" file on the VTOYEFI partition to set the default display mode to `text`. However, upon making a new installation with a different drive using version 1.0.40, I was unable to find "ventoy.json" or "ventoy_grub.cfg" on the new VTOYEFI partition. I also tried to remove the BIOS type from the branding, however those changes were reset even though other changes to grub.cfg were saved.

**Comment by ventoy** [comment]
ventoy.json and ventoy_grub.cfg  should be put in the 1st partition (ventoy partition) not the 32MB VTOYEFI partition.

Besides, when you make a install of ventoy (not update) the whole disk will be reformarted and all the data will lost.
When you update Ventoy, the data in the 1st partition (ventoy partition) will remain and all the data in VTOYEFI partition will lost.

The 32MB VTOYEFI partition is for Ventoy's internal use. So you should NOT put any files in it.

**Comment by andybot89** [comment]
I don't think that you understand what i'm saying. In previous versions, ventoy.json would be automatically placed in /ventoy/ on the 32MB VTOYEFI partition. however in version 1.0.40, this is not the case, and I would get "missing file" errors whenever I try to use an option relating to this file. Sorry for my late reply.

**Comment by xbps-install** [comment]
yeah after looking i found where it actually belongs. the reason I thought this is because `ventoy.json` was in /ventoy/ on the VTOYEFI partition in version 0.69, and I was confused as to where it moved after I updated to 1.0.40

**Comment by rawxdplay** [comment]
ventoy_grub.cfg not exist
What happened?
![360截图17980105839972](https://github.com/ventoy/Ventoy/assets/141987446/430e6fd4-a58c-41f2-a728-b296a9485f4a)

**Comment by sailinggalaxians** [comment]
Ping, I see this as well when pressing F6 for Exmenu.

**Comment by steve6375** [comment]
That file must be added by the end user (you)
It can contain grub2 commands and menu entries but you have to make it yourself.
Read the documentation on the website for more details about an external user menu
